C/services/core/core_management_api.cpp

```diff
@@ -18,6 +18,69 @@ using namespace rapidjson;
 
 CoreManagementApi *CoreManagementApi::m_instance = 0;
 
+/**
+ * Wrapper for "fake" registrer category interest
+ *
+ * TODO implement the missing functionality
+ * This method is just a fake returning a fixed id to caller
+ */
+void registerInterestWrapper(shared_ptr<HttpServer::Response> response,
+			     shared_ptr<HttpServer::Request> request)
+{
+	string payload("{\"id\" : \"1232abcd-8889-a568-0001-aabbccdd\"}");
+	*response << "HTTP/1.1 200 OK\r\nContent-Length: " << payload.length() << "\r\n"
+		  <<  "Content-type: application/json\r\n\r\n" << payload;
+}
+
+/**
+ * Easy wrapper for getting a specific service.
+ * It is called to get storage service details:
+ * example: GET /foglamp/service?name=FogLAMP%20Storage
+ *
+ * Immediate utility is to get the management_port of
+ * storage service when running tests.
+ * TODO fully implemtent the getService API call
+ */
+void getServiceWrapper(shared_ptr<HttpServer::Response> response,
+		       shared_ptr<HttpServer::Request> request)
+{
+
+	// Get QUERY STRING from request
+	string queryString = request->query_string;
+
+	size_t pos = queryString.find("name=");
+	if (pos != std::string::npos)
+	{
+		string serviceName = queryString.substr(pos + strlen("name="));
+		// replace %20 with SPACE
+		serviceName = std::regex_replace(serviceName,
+						 std::regex("%20"),
+						 " ");
+		ServiceRegistry* registry = ServiceRegistry::getInstance();
+		ServiceRecord* foundService = registry->findService(serviceName);
+		string payload;
+
+		if (foundService)
+		{
+			// Set JSON string with service details
+			foundService->asJSON(payload);
```

Comment:
The returned payload is not complete, it should have the uuid also.

Please add this as a todo.